src/main/java/io/th0rgal/oraxen/mechanics/provided/combat/spell/thor/ThorMechanicListener.java

```diff
@@ -36,7 +36,8 @@ public void onCall(PlayerInteractEvent event) {
         if (event.getAction() != Action.RIGHT_CLICK_AIR && event.getAction() != Action.RIGHT_CLICK_BLOCK) return;
         if (event.useItemInHand() == Event.Result.DENY) return;
         if (BlockHelpers.isInteractable(block) && event.useInteractedBlock() == Event.Result.ALLOW) return;
-        if (!ProtectionLib.canUse(player, location)) return;
+        //if (!ProtectionLib.canUse(player, location)) return;
+        if (!ProtectionLib.canUse(player, player.getTargetBlock(null, 50).getLocation())) return;
```

Comment:
Remove the comment and make a variable with the targetLock instead of reusing it here and later down